static/js/tools/stat_var/page.tsx

```diff
@@ -167,12 +166,130 @@ class Page extends Component<unknown, PageStateType> {
     );
   }
 
-  private updateHash(sv: string): void {
-    window.location.hash = `#${sv}`;
+  /**
+   * Fetches sources to display in dropdown menu.
+   */
+  private async fetchSources(): Promise<void> {
```

Comment:
i don't think you need the async & this should just return void because you're not actually returning the promise?

Reply:
done